core/src/epicli/data/common/defaults/configuration/feature-mapping.yml

```diff
@@ -109,6 +111,7 @@ specification:
       - logging
       - firewall
     single_machine:
+      - sshtunnel
```

Comment:
It's also need to execute in 'monitoring', `kubectl` is also used for prometheus.

Reply:
FIxed.